lib/gap_intelligence/models/merchant_pricing_trend.rb

```diff
@@ -0,0 +1,9 @@
+module GapIntelligence
+  class MerchantPricingTrend < Record
+    attributes :values, :headers
+
+    def initialize(attrs = {})
+      @attributes = attrs
+    end
```

Comment:
`@attributes = raw.fetch('attributes', raw)` will use `@raw` as a source for default value only, but will keep in a separate `attributes` instance variable